sbn/codingconv/CodingConventions.md

```diff
@@ -681,7 +681,7 @@ A similar recommendation still holds for code in implementation files
   ```
 
 
-**[F]** The use of `const_cast` and `reinterpret_cast` is **forbidden**.
+**[F]** The use of `const_cast` and `reinterpret_cast` is **forbidden** except when interfacing with broken library code, in which case a large and thorough comment is required.
```

Comment:
```suggestion
**[F]** The use of `const_cast` and `reinterpret_cast` is **forbidden** except when interfacing with broken external library code, in which case a large and thorough comment is required.
```
If the code is ours, we should fix it instead.

Reply:
agreed, that's what I tried to convey with just "library", but your way is more explicit.